src/main/java/frc/robot/joystickManager/JoystickManager.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.joystickManager;
+
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.Robot;
+import frc.joysticks.SmartJoystick;
+
+public class JoystickManager {
+
+	private final SmartJoystick[] joysticks;
+
+	private final Robot robot;
+
+	public JoystickManager(Robot robot) {
+		joysticks = new SmartJoystick[6];
```

Comment:
```suggestion
		joysticks = new SmartJoystick[MAGIC_NUMBER];
```
![image](https://github.com/user-attachments/assets/c44e36a8-d621-46c7-a67e-a773557b7cae)